crates/volta-layout/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "volta-layout"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Very minor nit: Since this is a purely internal crate, there's not much need to bump the version on changes 😄